src/dials/command_line/correlation_matrix.py

```diff
@@ -23,11 +23,14 @@
 
 phil_scope = iotbx.phil.parse(
     """\
-include scope dials.algorithms.correlation.analysis.phil_scope
+include scope dials.algorithms.correlation.analysis.working_phil
 
 seed = 42
```

Comment:
This seed, as well as the cosym seed, should be moved into the cosym common phil scope in algorithms/symmetry/cosym/__init__.py. We should use the common value of 230 that is used for the symmetry programs.

Reply:
seed removed from correlation_matrix and cosym commandline programs and put into the cosym __init__ phil scope